model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -48,6 +48,13 @@ data class OrtConfiguration(
      */
     val severeIssueThreshold: Severity = Severity.WARNING,
 
+    /**
+     * Enable the usage of project-local package-curations from the [RepositoryConfiguration]. If set to true apply
+     * package-curations from a local .ort.yml file. These package-curations are merged with package-curations from
+     * the command line option, while the one from .ort.yml take precedence.
+     */
+    val applyRepositoryPackageCurations: Boolean = false,
```

Comment:
Both the log statement in `AnalyzerCommand.kt` and the docs here use the term "enable". So how about also naming the property `enableRepositoryPackageCurations`?